plugins/woocommerce/includes/admin/wc-admin-functions.php

```diff
@@ -102,7 +103,7 @@ function wc_create_page( $slug, $option = '', $page_title = '', $page_content =
 	if ( $option_value > 0 ) {
 		$page_object = get_post( $option_value );
 
-		if ( $page_object && 'page' === $page_object->post_type && ! in_array( $page_object->post_status, array( 'pending', 'trash', 'future', 'auto-draft' ), true ) ) {
+		if ( $page_object && 'page' === $page_object->post_type && ! in_array( $page_object->post_status, array( OrderStatus::PENDING, OrderStatus::TRASH, 'future', OrderStatus::AUTO_DRAFT ), true ) ) {
```

Comment:
Similar to an earlier note, this works, but I'm not sure if it's appropriate as we're examining post statuses (for pages), rather than order statuses.